app/services/cfe/base_service.rb

```diff
@@ -1,12 +1,13 @@
 module Cfe
   class BaseService
-    def self.call(session_data, payload)
-      new(session_data, payload).call
+    def self.call(session_data, payload, early_eligibility = nil)
```

Comment:
Think this can only be nil if the feature flag is false - think it might be better being explicitly passed in if so, otherwise it might get left in the codebase when the feature flag is removed.

Reply:
This can be nil when the CFE service is called at the end of the journey -  I added an optional argument for when we call the CFE service early, when we have less information to send over but can still provide an ineligible outcome. This is because currently I am using the presence of `early_eligibility` as a guard clause in the CFE service, so we know we can ignore sending certain bits of info to CFE. But this was a starting point so maybe there is a better refactor!

Reply:
My suggestion was just to move the default up - so making the parameter nil explicitly in the caller